packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css

```
@@ -1,15 +1,17 @@
 @import 'settings/colors';
 @import 'settings/typography';
 @import 'settings/transitions';
+@import '@contentful/forma-36-tokens/dist/css/spacing.css';
```

This should work and will keep things more consistent -

```suggestion
@import 'settings/dimensions';
```